bug/op_edit_comment.go

```diff
@@ -72,10 +72,12 @@ func (op *EditCommentOperation) Apply(snapshot *Snapshot) {
 
 	// Updating the corresponding comment
 
-	for i := range snapshot.Comments {
-		if snapshot.Comments[i].Id() == commentId {
-			snapshot.Comments[i].Message = op.Message
-			snapshot.Comments[i].Files = op.Files
+	//TODO use snapshot.SearchComment instead of iteration
+	//TODO put *editComment* in snapshot or add *edit* to Comment interface?
```

Comment:
Implementing an *editComment* within snapshot seems to be wrong. It would require multiple functions to also alter comment attachments etc. This would artificially blow up snapshot in size.

Reply:
Use snapshot::UpdateComment. This Snapshot doesn't have to provide *edit* methods for each Comment attribute.
Note, that this might still be subject to change.